Cython/Includes/cpython/datetime.pxd

```diff
@@ -119,21 +193,41 @@ cdef extern from "datetime.h":
     int PyDateTime_DATE_GET_MINUTE(object o)
     int PyDateTime_DATE_GET_SECOND(object o)
     int PyDateTime_DATE_GET_MICROSECOND(object o)
+    int PyDateTime_DATE_GET_FOLD(object o)
+    object PyDateTime_DATE_GET_TZINFO(object o)
 
     # Getters for time (C macros).
     int PyDateTime_TIME_GET_HOUR(object o)
     int PyDateTime_TIME_GET_MINUTE(object o)
     int PyDateTime_TIME_GET_SECOND(object o)
     int PyDateTime_TIME_GET_MICROSECOND(object o)
+    int PyDateTime_TIME_GET_FOLD(object o)
+    object PyDateTime_TIME_GET_TZINFO(object o)
 
     # Getters for timedelta (C macros).
     int PyDateTime_DELTA_GET_DAYS(object o)
     int PyDateTime_DELTA_GET_SECONDS(object o)
     int PyDateTime_DELTA_GET_MICROSECONDS(object o)
 
+    # Constructors
+    object PyDateTime_FromDateAndTimeAndFold(
+        int year, int month, int day, int hour, int min, int sec, int usec, int fold
+    )
+    object PyTime_FromTimeAndFold(int hour, int minute, int second, int usecond, int fold)
+
+    # We don't need this first one
+    # object PyTimeZone_FromOffset(int offset)
+    object PyTimeZone_FromOffsetAndName(int offset, str name)
+
+    # Constructors for the DB API
+    object PyDateTime_FromTimeStamp(object args)
+    object PyDate_FromTimeStamp(object args)
+
     # PyDateTime CAPI object.
     PyDateTime_CAPI *PyDateTimeAPI
 
+    PyObject* PyDateTime_TimeZone_UTC
```

Comment:
I think we can declare this as `object`. 
```suggestion
    object PyDateTime_TimeZone_UTC  # requires Py3.7+ !
```

Reply:
No we cannot, see the following error:

```
------------------------------------------------------------
...
    object PyDate_FromTimeStamp(object args)

    # PyDateTime CAPI object.
    PyDateTime_CAPI *PyDateTimeAPI

    object PyDateTime_TimeZone_UTC  # requires Py3.7+ !
          ^
------------------------------------------------------------

datetime.pxd:223:11: Python object cannot be declared extern

Reply:
Right. Then the `get_utc()` function has its value.